core/src/main/scala/backbone/consumer/ReceiveSettings.scala

```diff
@@ -1,22 +1,99 @@
 package backbone.consumer
 
+import java.time.Duration
+import java.{util => ju}
+
+import akka.stream.alpakka.sqs.{MessageAttributeName, MessageSystemAttributeName}
```

Comment:
we might want to remove the ReceiveSettings class entirely and replace it with SqsSourceSettings from backbone directly. Right now this is more or less a copy, and each value needs to be mapped in the Consumer. Also I'm using alpakka sqs classes here as it would be quite some files to copy + map them again.

What do you think?

Reply:
I think receiving them interely is a good idea.